examples/c/sockfilter.c

```diff
@@ -90,6 +98,21 @@ int main(int argc, char **argv)
 	struct ring_buffer *rb = NULL;
 	struct sockfilter_bpf *skel;
 	int err, prog_fd, sock;
+	int argp;
+	char *interface = "lo";
+
+	while ((argp = getopt(argc, argv, "hi:")) != -1) {
```

Comment:
see bootstrap example which uses argp framework for passing arguments. Let's use just one way (argp).